Dockerfile

```diff
@@ -12,7 +12,16 @@ WORKDIR /app
 # build typed-ast in separate stage because it requires gcc and libc-dev
 FROM base AS python-deps
 RUN apt-get update -qqy && apt-get install -qqy gcc libc-dev
-COPY requirements.txt ./
+
+# install poetry
+RUN pip install --no-cache-dir --quiet poetry
```

Comment:
[preferred way of installing](https://python-poetry.org/docs/#installing-with-the-official-installer) `poetry` is 
```shell
curl -sSL https://install.python-poetry.org | python - --version <<poetry_version>>
```